### PR TITLE
Web-console: add resizable split screen layout to tasks and servers views 

### PR DESCRIPTION
**Pull request by mcbrewster**
Screen Shot 2019-05-21 at 12 37 18 PM

React-splitter-layout allows for the split screen views to be dragged and resized. When the user finishes dragging the current size of the secondary pane is stored in local storage so the layout remains the same if the page is refreshed. If there is no size value in local storage the view will automatically be set to 60/40. 

Additionally the navigation buttons now follow the pane when resized.

### CODE REVIEW

File: web-console/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "@blueprintjs/core": "^3.15.1",
+    "@types/react-splitter-layout": "^3.0.0",
```
Comment (vogievetsky) [inline web-console/package.json]: the types are dev dependancies

Comment (mcbrewster) [reply web-console/package.json]: fixed!

**Comment by vogievetsky** [comment]
Could you please make the splitter match the style that it had prior to this PR?

**Comment by vogievetsky** [comment]
Thank you for all the changes. This is good to go 👍

**Pull request merged**